lib/simulation/openfisca/mapping/index.js

```diff
@@ -39,17 +39,20 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
     }
 
     var ressourcesByType = _.groupBy(src.ressources, 'type');
-    _.forEach(schema, function(definition, destKey) {
-        var srcKeys, fn;
-        if (_.isString(definition)) {
-            srcKeys = [definition];
-        } else if(_.isArray(definition)) {
-            srcKeys = definition;
-        } else {
-            srcKeys = [definition.src];
-            fn = definition.fn;
+
+    _.forEach(schema, function(definitions, destKey) {
+        if (! _.isArray(definitions)) {
+            definitions = [definitions];
         }
-        _.forEach(srcKeys, function(srcKey) {
+
+        _.forEach(definitions, function(definition) {
+            var fn, srcKey;
+            if (_.isString(definition)) {
+            srcKey = definition;
+            } else {
+                srcKey = definition.src;
+                fn = definition.fn;
```

Comment:
This could be simpler: `srcKey = definition.src || definition`, and `fn = definition.fn`: if it is not set, it will be `undefined`, just like when you write `var fn` :)